webextensions/sidebar/subpanel.js

```diff
@@ -194,6 +194,10 @@ async function applyProvider(id) {
       }
     }
 
+    const url = new URL(provider.subPanel.url);
+    url.searchParams.set('tst-windowId', mTargetWindow);
```

Comment:
The prefix `tst-` looks unnecessary, I think `windowId` is enough reasonable.

Reply:
\yeah, there should be very little reason for anyone to want to have a conflicting parameter ...